tests/stubtest_allowlists/win32-py312.txt

```diff
@@ -5,7 +5,6 @@ asyncio.windows_events.IocpProactor.finish_socket_func
 asyncio.windows_events.IocpProactor.recvfrom_into
 msvcrt.GetErrorMode
 ntpath.isdir
-os.EX_OK
 os.get_blocking
 os.listdrives
 os.listmounts
```

Comment:
Looks like a bunch of functions were added to the `os` module on Windows in Python 3.12, that we haven't added yet in typeshed. If you fancy tackling that in a followup PR, it would be really valuable!

Reply:
yeah sure, I'd love to.